src/pymoca/parser.py

```diff
@@ -346,60 +347,42 @@ def exitExpression_if(self, ctx: ModelicaParser.Expression_ifContext):
         conditions = all_expr[:-1:2]
         expressions = all_expr[1::2] + all_expr[-1:]
 
-        self.ast[ctx] = ast.IfExpression(
-            conditions=conditions,
-            expressions=expressions)
+        self.ast[ctx] = ast.IfExpression(conditions=conditions, expressions=expressions)
 
     def exitExpr_primary(self, ctx: ModelicaParser.Expr_primaryContext):
         self.ast[ctx] = self.ast[ctx.primary()]
 
     def exitExpr_add(self, ctx: ModelicaParser.Expr_addContext):
         self.ast[ctx] = ast.Expression(
-            operator=ctx.op.text,
-            operands=[self.ast[e] for e in ctx.expr()]
+            operator=ctx.op.text, operands=[self.ast[e] for e in ctx.expr()]
         )
 
     def exitExpr_exp(self, ctx: ModelicaParser.Expr_expContext):
         self.ast[ctx] = ast.Expression(
-            operator=ctx.op.text,
-            operands=[self.ast[e] for e in ctx.primary()]
+            operator=ctx.op.text, operands=[self.ast[e] for e in ctx.primary()]
         )
 
     def exitExpr_mul(self, ctx: ModelicaParser.Expr_mulContext):
         self.ast[ctx] = ast.Expression(
-            operator=ctx.op.text,
-            operands=[self.ast[e] for e in ctx.expr()]
+            operator=ctx.op.text, operands=[self.ast[e] for e in ctx.expr()]
         )
 
     def exitExpr_rel(self, ctx: ModelicaParser.Expr_relContext):
         self.ast[ctx] = ast.Expression(
-            operator=ctx.op.text,
-            operands=[self.ast[e] for e in ctx.expr()]
+            operator=ctx.op.text, operands=[self.ast[e] for e in ctx.expr()]
         )
 
     def exitExpr_not(self, ctx: ModelicaParser.Expr_notContext):
-        self.ast[ctx] = ast.Expression(
-            operator='not',
-            operands=[self.ast[ctx.expr()]]
-        )
+        self.ast[ctx] = ast.Expression(operator="not", operands=[self.ast[ctx.expr()]])
 
     def exitExpr_and(self, ctx: ModelicaParser.Expr_andContext):
-        self.ast[ctx] = ast.Expression(
-            operator='and',
-            operands=[self.ast[e] for e in ctx.expr()]
-        )
+        self.ast[ctx] = ast.Expression(operator="and", operands=[self.ast[e] for e in ctx.expr()])
 
     def exitExpr_or(self, ctx: ModelicaParser.Expr_orContext):
-        self.ast[ctx] = ast.Expression(
-            operator='or',
-            operands=[self.ast[e] for e in ctx.expr()]
-        )
+        self.ast[ctx] = ast.Expression(operator="or", operands=[self.ast[e] for e in ctx.expr()])
 
     def exitExpr_signed(self, ctx: ModelicaParser.Expr_signedContext):
-        self.ast[ctx] = ast.Expression(
-            operator=ctx.op.text,
-            operands=[self.ast[ctx.expr()]]
-        )
+        self.ast[ctx] = ast.Expression(operator=ctx.op.text, operands=[self.ast[ctx.expr()]])
```

Comment:
Well black prefers less readable with these `Expr`*, haha. I guess this shows a human can sometimes make better decisions than hard-wired formatting rules.

Reply:
I think we can force what you want by adding a comma after the `operands=[self.ast[ctx.expr()]]` line